app/flatpak-builtins-add-remote.c

```diff
@@ -361,8 +362,23 @@ flatpak_builtin_add_remote (int argc, char **argv,
     return FALSE;
 
   /* We can't retrieve the extra metadata until the remote has been added locally, since
-     ostree_repo_remote_fetch_summary() works with the repository's name, not its URL. */
-  return flatpak_dir_update_remote_configuration (dir, remote_name, cancellable, error);
+     ostree_repo_remote_fetch_summary() works with the repository's name, not its URL.
+     Don't propagate IO failed errors here because we might just be offline - the
+     remote should already be usable. */
+  if (!flatpak_dir_update_remote_configuration (dir, remote_name, cancellable, &local_error))
+    {
+      if (g_error_matches (local_error, G_IO_ERROR, G_IO_ERROR_FAILED))
+        {
+          g_warning (_("Could not update extra metadata for '%s': %s\n"), remote_name, local_error->message);
```

Comment:
It's very unusual to mark warnings for translation, and no other warnings are marked as such in the flatpak code base at the moment. Please change it to use a simple string.